spec/cspec/c/gen-config-thy.py

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021, Proofcraft Pty Ltd
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+#
+# This script generates a theory Kernel_Config.thy from the file gen_config.h,
+# which in turn is produced by the kernel build system, containing kernel
+# configuration options.
+#
+# The script makes the following assumptions about gen_config.h:
+#
+#   - only lines that start with "#define" are config options that are used
+#   - the value of the config option is separated from its name by two spaces
+#   - there is an optional comment after the value, separated by two spaces
```

Comment:
perhaps an example? proverbial picture is worth a thousand words?

Reply:
Yes, good idea. Will do.

- [ ] add example